eng/Versions.props

```diff
@@ -65,6 +65,9 @@
     <MicrosoftDotNetGenAPIPackageVersion>5.0.0-beta.20171.1</MicrosoftDotNetGenAPIPackageVersion>
     <!-- Packages from dotnet/roslyn -->
     <MicrosoftNetCompilersToolsetPackageVersion>3.6.0-3.20170.19</MicrosoftNetCompilersToolsetPackageVersion>
+    <!-- Packages from dotnet/roslyn-sdk -->
+    <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20178.1</MicrosoftCodeAnalysisTestingVersion>
+    <MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitPackageVersion>$(MicrosoftCodeAnalysisTestingVersion)</MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitPackageVersion>
```

Comment:
📝 This package is only used in testing. I do not understand how to make it work since `<PackageReference>` is broken in this repository.

Reply:
@sharwell - this repo uses dotnet/arcade - so most things in this file are updated in an automated way. This can go in the other `ItemGroup` - https://github.com/dotnet/aspnetcore/blob/fa48c73c3a2de3f9db84c38310e92a64f27d5c3f/eng/Versions.props#L173

Reply:
Yeah, sorry we do strange things for strange reasons. You should be able to just use it as `<Reference Include="PackageName" />`.

Shout if you're stuck here and I can help you.